marklogic-data-hub/src/main/java/com/marklogic/hub/deploy/commands/DeployQueryOptionsCommand.java

```diff
@@ -0,0 +1,22 @@
+package com.marklogic.hub.deploy.commands;
```

Comment:
Pinging @rjrudin, @kkanthet, do we need to add the copyright information anymore?
Some files have it and some don't.

Reply:
We should have it when adding new files.